### PR TITLE
feat: Modal and Toast

### PR DESCRIPTION
**Pull request by maitrakhatri**
# features added
- Modal
- Toast

# screenshots
Screenshot 2022-03-24 at 16-12-51 gozo UI Modal
Screenshot 2022-03-24 at 16-13-12 gozo UI Snackbar

### CODE REVIEW

File: CSS/Components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.snackbar {
+    background-color: rgb(51, 51, 51);
+    border-radius: .5rem;
+    display: inline-flex;
+    align-items: center;
+    gap: 1rem;
+    padding: .15rem .75rem;
+    color: white;
```
Comment (anupkgurung) [inline CSS/Components/toast/toast.css]: use hex and color should be in variables most of the time.

File: CSS/Components/toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+a, i {
+    padding: 0.25rem;
+}
+
+a:hover, i img:hover {
+    cursor: pointer;
+    background-color:rgba(128, 128, 128, 0.26);
+}
```
Comment (anupkgurung) [inline CSS/Components/toast/toast.css:23]: try to use hex and variables

**Review by anupkgurung — APPROVED** [review]
try to use hex and variables. rest look good !!

File: CSS/Components/toast/toast.css (hunk 1 above)
Comment (maitrakhatri) [reply CSS/Components/toast/toast.css]: done

File: CSS/Components/toast/toast.css (hunk 2 above)
Comment (maitrakhatri) [reply CSS/Components/toast/toast.css:23]: done

**Review by SanketDhabarde — APPROVED** [review]
- code looks clean and simple
- liked that you used proper semantics such as `<aside>`, `<main>`, etc. whenever needed
- folder structure looks good

File: CSS/Components/modal/modal.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.modal .card-body {
+    border-bottom: 0.063rem solid #000000;
+}
+
+.modal .card-action {
+    padding: .75rem;
```
Comment (SanketDhabarde) [inline CSS/Components/modal/modal.css:6]: You can create padding margin utility classes. just suggestion.

Comment (maitrakhatri) [reply CSS/Components/modal/modal.css:6]: yes I'm planning to add them

**Pull request merged**